server.js

```diff
@@ -0,0 +1,153 @@
+
+// var {google} = require('googleapis');
+// var fs = require('fs');
+// var readline = require('readline');
+//database
+// mongoose.Promise = global.Promise;
+// mongoose.connect(process.env.MONGO_DB, {useMongoClientCient: true});
+var db = mongoose.connection;
+db.once('open', function() {
+    console.log('DB connected!');
+});
+db.on('error', function (err) {
+    console.log('DB ERROR : ', err);
+});
+
+//Middlewares
+var app = express();
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(express.static(__dirname + '/public'));
+
+// app.use(function (req, res, next){
+//     res.header('Access-Control-Allow-Origin');
+//     res.header('Access-Cotrol-Allow-Methods', 'GET, POST, PUT, DELETE');
+//     res.header('Access-Control-Allow-Headers', 'content-type');
+//     next();
+// });
+
+
+
+var doc = new googleSpreadsheet('1KpgW0E_Oa_BwoFxWIS6eLrgho6WaBzNaoU6gC4rdmGI');
+var sheet;
+
+async.series([
+    function setAuth(step) {
+        var creds_json = {
+            client_email: 'jaram-groupware@jaram-groupware.iam.gserviceaccount.com',
+            private_key: '-----BEGIN PRIVATE KEY-----\nMIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCgZ3dRfJhEWc+e\nq9oSB34fh1rzrR9bnnOQy9ccN+McRXFTYCZxHC8M20BNpSjqZzm85MPMqq7j5uek\nfKEM2TqFpn7i1WKzJMLAZNoQyQpo+s3IpSMCGmVLRp4MNK3hrtrewljNGrgNpOt8\n4dDRMSE9ZBZJUxlWaTwrcUmmG0oW8OgvSSniYfOXqjVV4AFZ5sA3Qnl7XtUlfjC3\niRewavJiz8q9NQaDfpWbi1yYxRl4SP+mQwqbe6xkOu9tTfOi4fhjh8UU+BsOid//\nVbJRIZtG8Ch40j4LIu5qd/T6lLRf2IqqV5hLgwcWD6yom3JzhwNI3vbhLCxGHa2l\nbSnh6J2pAgMBAAECggEAEj/N70CV0Z+cLkkOrY1vrSZI/pSRjuq8lYUzdduVOagG\n6ChLcgP1sGiYwhy1G9IJsMggJKlomLeHtDWVBxwHJfcsKXeDMVl7Kh4Z7IigPWhc\nwDlVfc2Yp6ii85riC4f0ER2bB8lvQOd5/pfHGlMUx+Iat6s+JJAizy3Q5haQZoKX\njslTubm4yHaMRs8jUbfMDJr9Cjzo7nDuSWl9jpIENc2+Du/YlycAlQoQPFVeEmjx\n0P0BDODHCCyAcQWHyNbtXXagZlpzloD8dgt+bg1XlFyYaM7jUKLJBAog5VT5mTnG\n4uNfpcrvaMZzRKlfNARsx6FSwi9sgsRDGV5hyvgKTwKBgQDVXfPQoATZrEeVkLvG\nnYezbr3A9NMzi88GzGNXZqNudHRGLjUu46U0crz6cmfd6kEezoxXx4SyTqumJ1eX\nh9l7QnrshlC/IDtu3nPQrkLFYQummf4+zy2Ix3Cmdu0kq9CXLggNkDAZbQ0yBGoo\noa+tYYWesJTLcvz0lddqycznxwKBgQDAdGI/E+fm4CnxH6kKVec4iKrX/Zmse80c\nwcEIja8K8brFUVshNTuxThIb9SGkvSpBinlcCpoOFd+PlTHZn6toc80WL/0TcSC6\nNogPDYOSPdmtVEHg9N9Qp0Ccw0elX1+Yo0yrB1HD+LQmeTUQY3EG98wTSunb4fZR\nvVsNe9qvDwKBgQCqyuYcHbtEMq+qfYyR1A+IBiFGj7NwE33Ix1rN3477H6TjeKGB\nwx+++AllwKZ78MPn0RZuPXtu76JfAicPWOMx02lqY6nHFBn/CXpTlOpF7QTSe3oU\n9OpSDrORzKjPLelu2p3/nWejMrPZUcdYdvSVKPc4q8ivgoARmNAy3w5yiQKBgQCU\n5XZddvkkFqpcczJFXNDXXGVRVlNerymx5fWCuJuEzCDEKH1o49abBtvb5XtyCSh4\nLLpYC5vE6gdc/wDibjYfSfiSAe666/t2FlqNytbQfim7gES31rwvmjbTEgKMX42G\nelEGwVLxW6mFWqFDXKxFx9GL25mzFM+NzgJ0HC7E4QKBgH7mbz3U8ifEzOKfOqak\njNIrSmkXpMUYZtUFqk2Flai6mG1IWk9tOtZ0Y721DqEDqhJAj8kOEUaMYLtsKQuY\nMapmSIeIaeEzo8tWYTH/j2Wu0u/W07c7+Xw8u0ONDHztV2/KB8c6IYhWlWm6T6cC\nrfqrXBbyBTWZvvBnH0LGuSik\n-----END PRIVATE KEY-----\n'
```

Comment:
I think it is a 'private key' so that you need to remove the key.

Reply:
I've deleted it.